src/ontology/mondo-edit.obo

```diff
@@ -263928,7 +263946,8 @@ xref: NCIT:C122782 {source="MONDO:equivalentTo"}
 xref: OMIM:607015 {source="Orphanet:93476", source="MONDO:equivalentTo", source="Orphanet:93476/e"}
 xref: Orphanet:579 {source="OMIM:607015"}
 xref: Orphanet:93476 {source="MONDO:equivalentTo", source="OMIM:607015"}
-xref: SCTID:73123008 {source="MONDO:equivalentTo"}
+xref: SCTID:26745009 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
+xref: SCTID:73123008 {source="https://orcid.org/0009-0002-1597-2198"}
```

Comment:
Change back to `source="MONDO:equivalentTo"`

Reply:
this needs to be reverted to fix the proxy merge issue and add curator Orcid, https://orcid.org/0009-0002-1597-2198